cpp/src/c_api/replicate_edgelist.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cugraph_c/algorithms.h>
+#include <cugraph_c/graph_generators.h>
+
+#include <c_api/abstract_functor.hpp>
+#include <c_api/capi_helper.hpp>
+#include <c_api/graph.hpp>
+#include <c_api/induced_subgraph_result.hpp>
+#include <c_api/resource_handle.hpp>
+#include <c_api/utils.hpp>
+
+#include <cugraph/algorithms.hpp>
+#include <cugraph/detail/shuffle_wrappers.hpp>
+#include <cugraph/detail/utility_wrappers.hpp>
+#include <cugraph/graph_functions.hpp>
+
+namespace {
+
+template <typename vertex_t, typename weight_t>
```

Comment:
I assume this is leftover from debugging and should be deleted.

Reply:
resolved